parcels/field.py

```diff
@@ -444,17 +438,7 @@ def from_netcdf(
         netcdf_engine = kwargs.pop("netcdf_engine", "netcdf4")
         gridindexingtype = kwargs.get("gridindexingtype", "nemo")
 
-        indices = {} if indices is None else indices.copy()
-        for ind in indices:
-            if len(indices[ind]) == 0:
-                raise RuntimeError(f"Indices for {ind} can not be empty")
-            assert np.min(indices[ind]) >= 0, (
-                "Negative indices are currently not allowed in Parcels. "
-                + "This is related to the non-increasing dimension it could generate "
-                + "if the domain goes from lon[-4] to lon[6] for example. "
-                + "Please raise an issue on https://github.com/OceanParcels/parcels/issues "
-                + "if you would need such feature implemented."
-            )
+        indices = {}  # TODO Nick: Cleanup
```

Comment:
Will you stil clean this up in this PR? Or in a next one?

Reply:
Forgot to remove this comment. Really struggled to clean up any further than here :/

Reply:
done in 0f89a3d044a33c07ba5fcd241a52f8d077aa7674 (separate PR to avoid lots of rebasing)